drivers/sx127x/sx127x_getset.c

```diff
@@ -446,7 +446,7 @@ uint8_t sx127x_get_bandwidth(const sx127x_t *dev)
     return dev->settings.lora.bandwidth;
 }
 
-inline void _low_datarate_optimize(sx127x_t *dev)
+static inline void _low_datarate_optimize(sx127x_t *dev)
```

Comment:
Then also remove `inline` which is useless.

Reply:
makes sense. I will